cmd/influx_inspect/main.go

```diff
@@ -49,7 +48,7 @@ func main() {
 			os.Exit(1)
 		}
 		cmdInfo(path)
-	case "dumptsm":
+	case "dumptsm", "dumptsmdev":
```

Comment:
Can you log a warning that `dumptsmdev` is deprecated and to use `dumptsm` going forward?